ci-operator/step-registry/openshift/microshift/clusterbot/prepare-host/openshift-microshift-clusterbot-prepare-host-commands.sh

```diff
@@ -42,8 +42,12 @@ if ! sudo subscription-manager status >&/dev/null; then
 		--activationkey="$(cat /tmp/subscription-manager-act-key)"
 fi
 
-sudo dnf clean all
-hash git || sudo dnf install -y git-core
+PKGS="git-core make"
+sudo dnf clean all --enablerepo=\*
+if ! sudo dnf install -y ${PKGS}; then
+	sudo rm -rf /var/cache/dnf
+	sudo dnf install -y ${PKGS}
+fi
```

Comment:
I would try something similar to dnf_retry here: 
https://github.com/openshift/microshift/blob/main/scripts/devenv-builder/configure-vm.sh#L30

Deleting the directory should be no different than clean